source/configuration_manual/authentication/lua_based_authentication.rst

```diff
@@ -29,16 +29,47 @@ For details about Dovecot Lua, see :ref:`lua`.
 When used in authentication, additional module dovecot.auth is added, which
 contains constants for passdb and userdb.
 
+Lua specific passdb/userdb settings
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. dovecot_core:setting:: userdb_lua_path
```

Comment:
isn't it enough to have just lua_path?

Reply:
If the lua_path setting was moved to lib-lua, I don't think it should be even listed here. Although could be linked to somewhere in the page.